kuma/wiki/jinja2/wiki/ckeditor_config.js

```diff
@@ -90,6 +90,41 @@
     // Don't use HTML entities in the output except basic ones (config.basicEntities).
     config.entities = false;
 
+    // Enable the spell checker by default
+    config.scayt_autoStartup = true;
```

Comment:
I'm considering a run-time switch to turn this on or off, because I'm worried we will hit an API limit with SKAYT.